mirror/mirror.go

```diff
@@ -39,6 +74,17 @@ func get(url string) ([]byte, error) {
 	return body, err
 }
 
+func getUrl(serverPrefix, path string) string {
+	scheme := "https"
+	if serverPrefix == "localhost" ||
+		strings.HasPrefix(serverPrefix, "localhost:") ||
+		strings.HasPrefix(serverPrefix, "localhost/") {
+		scheme = "http"
+	}
+
+	return scheme + "://" + serverPrefix + "/mtc/v1/" + path
```

Comment:
Currently we have the extra `/static/` in the path for static assets, but probably best to remove that.